techniques/ios/MASTG-TECH-0056.md

```diff
@@ -3,71 +3,82 @@ title: Installing Apps
 
-On Linux and also macOS, you can alternatively use [libimobiledevice](https://www.libimobiledevice.org/ "libimobiledevice"), a cross-platform software protocol library and a set of tools for native communication with iOS devices. This allows you to install apps over a USB connection by executing ideviceinstaller. The connection is implemented with the USB multiplexing daemon [usbmuxd](https://www.theiphonewiki.com/wiki/Usbmux "Usbmux"), which provides a TCP tunnel over USB.
+<img src="Images/Techniques/0056-Sideloadly.png" width="400px" />
 
-The package for libimobiledevice will be available in your Linux package manager. On macOS you can install libimobiledevice via brew:
-
-```bash
-brew install libimobiledevice
-brew install ideviceinstaller
-```
+## libimobiledevice
 
-If you have any issues, try installing the libraries from source, as the precompiled version may be outdated.
+On Linux and also macOS, you can alternatively use @MASTG-TOOL-0126. This allows you to install apps over a USB connection by executing `ideviceinstaller`. The connection is implemented with the USB multiplexing daemon [usbmuxd](https://www.theiphonewiki.com/wiki/Usbmux "Usbmux"), which provides a TCP tunnel over USB.
 
-After the installation you have several new command line tools available, such as `ideviceinfo`, `ideviceinstaller` or `idevicedebug`. Let's install and debug the @MASTG-APP-0028 app with the following commands:
+Let's install the @MASTG-APP-0028 app with the following command:
 
 ```bash
-# The following command will show detailed information about the iOS device connected via USB.
-$ ideviceinfo
-# The following command will install the IPA to your iOS device.
-$ ideviceinstaller -i iGoat-Swift_v1.0-frida-codesigned.ipa
+$ ideviceinstaller -i Uncrackable.ipa
 ...
 Install: Complete
-# The following command will start the app in debug mode, by providing the bundle name. The bundle name can be found in the previous command after "Installing".
-$ idevicedebug -d run OWASP.iGoat-Swift
 ```
 
+## Filza
+
+@MASTG-TOOL-0128 allows you to install an IPA file which is already located on your device. You can use either `scp` or airdrop to copy the IPA file to your device, after which you can simply navigate to the IPA file on your file system and click the `Install` button in the top right corner.
+
 ## ipainstaller
 
-The IPA can also be directly installed on the iOS device via the command line with [ipainstaller](https://github.com/autopear/ipainstaller "IPA Installer"). After copying the file over to the device, for example via scp, you can execute ipainstaller with the IPA's filename:
+The IPA can also be directly installed on the iOS device via the command line with [ipainstaller](https://github.com/autopear/ipainstaller "IPA Installer"). Naturally, this requires a jailbroken device, as otherwise you cannot SSH into the device. After copying the file over to the device, for example via scp, you can execute ipainstaller with the IPA's filename:
```

Comment:
```suggestion
The IPA can also be directly installed on the iOS device via the command line with [ipainstaller](https://github.com/autopear/ipainstaller "IPA Installer"). Naturally, this requires a jailbroken device, as otherwise you cannot SSH into the device. After copying the file over to the device, for example via scp or [AirDrop](https://support.apple.com/en-us/119857), you can execute `ipainstaller` with the IPA's filename:
```